e2e/test/iothub/service/IoTHubServiceProxyE2ETests.cs

```diff
@@ -88,17 +88,29 @@ private async Task JobClient_ScheduleAndRunTwinJob(HttpTransportSettings httpTra
                 Tags = new TwinCollection(),
             };
             twin.Tags[JobTestTagName] = JobDeviceId;
-
-            using var jobClient = JobClient.CreateFromConnectionString(s_connectionString, httpTransportSettings);
+            IotHubServiceClientOptions options = new IotHubServiceClientOptions();
+            options.Proxy = httpTransportSettings.Proxy;
```

Comment:
Use an initializer list.

```suggestion
            IotHubServiceClientOptions options = new IotHubServiceClientOptions
            {
                Proxy = httpTransportSettings.Proxy,
            }
```